hubspot_commerce.module

```diff
@@ -0,0 +1,46 @@
+<?php
```

Comment:
**CHANGE REQUEST**

Let's rename the module to `commerce_hubspot`, like all modules that are part of the Drupal Commerce ecosystem.

I have created a sandbox: https://www.drupal.org/sandbox/krystalcode/3039613, let's create an issue for this PR there.